src/repl.zig

```diff
@@ -163,11 +166,76 @@ pub fn repl(allocator: std.mem.Allocator) !void {
     var previous_parser_globals = try parser.globals.clone();
     var previous_globals = try vm.globals.clone();
     var previous_type_registry = try gc.type_registry.registry.clone();
+
+    var interrupted: i16 = 0;
+    var executed = std.ArrayList(u8).init(allocator);
+    defer executed.deinit();
+
     while (true) {
         const read_source = ln.linenoise("> ");
+
+        // quick check to prevent segfault.
+        const fmt = std.fmt.allocPrint(allocator, "{any}", .{read_source}) catch continue;
```

Comment:
Quite a lot of work here to do a simple thing. The issue was actually that I forgot the `?` in `lineoise` return type:

Here's the fix -> https://github.com/buzz-language/buzz/commit/fd9e5a4624745ac78db5c3bb4ae22005b7b4e1d9